include/cape_ros/cape_ros_node.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * This file is part of the cape_ros.
+ * Copyright (c) 2019 Avidbots Corp.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CAPE_ROS_CAPE_ROS_NODE_H
+#define CAPE_ROS_CAPE_ROS_NODE_H
+
+#include <ros/ros.h>
+#include <sensor_msgs/CameraInfo.h>
+#include <sensor_msgs/Image.h>
+#include <cape_ros/Planes.h>
+#include <cv_bridge/cv_bridge.h>
+#include <cape_ros/cape.h>
+#include <opencv2/opencv.hpp>
+
+class CapeRosNode
+{
+public:
+  CapeRosNode(ros::NodeHandle nh);
+
+  void simpleProjectPointCloud(cv::Mat& X_, cv::Mat& Y_, cv::Mat& Z, Eigen::MatrixXf& cloud_array_,
+                               const double& depth_cutoff);
+
+  void organizePointCloudByCell(Eigen::MatrixXf& cloud_in, Eigen::MatrixXf& cloud_out, cv::Mat& cell_map_);
```

Comment:
Same idea here. This does not match the function in the source file. `cell_map_ `should be `cell_map`